### PR TITLE
Add issue templates: bug report and feature request

### PR DESCRIPTION
**Pull request by Mariatta**
In all of them, mention that this tracker is for python.org.

### CODE REVIEW

File: .github/issue_template/custom.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+If you're looking to file an issue with CPython itself, please go to
+https://bugs.python.org
+
+Issues related to Python's documentation (https://docs.python.org) can
+also be filed in https://bugs.python.org, by selecting the "Documentation" component.
+--->
+
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
```
Comment (berkerpeksag) [inline .github/issue_template/custom.md]: Do we need this template to fallback to the regular ["new issue"](https://github.com/python/pythondotorg/issues/new) page? Or will GitHub show the "Don’t see your issue here? Open a regular issue." text by default even if we don't create `custom.md`?

screenshot from 2018-07-13 02 14 13

Comment (Mariatta) [reply .github/issue_template/custom.md]: Without `custom.md` the regular issue will still show up, so I guess it's not needed :)

Comment (Mariatta) [reply .github/issue_template/custom.md]: I'll remove custom.

**Review by berkerpeksag — APPROVED** [review]

**Comment by berkerpeksag** [comment]
Thanks!

**Pull request merged**